packages/plugins/i18n/admin/src/components/LocaleListCell.tsx

```diff
@@ -9,37 +9,24 @@ import { useGetLocalesQuery } from '../services/locales';
 interface LocaleListCellProps {
   documentId: string;
   collectionType: string;
+  localizations: { locale: string }[];
   locale: string;
   model: string;
```

Comment:
Should you remove those other unused props from this type? Or will they be passed upstream even if we don't use them?

Reply:
absolutely!